src/main.cpp

```diff
@@ -1021,6 +1037,7 @@ int main(int argc, char **argv) {
     case CmdTargets:
         return print_target_list(stdout);
     case CmdInvalid:
-        return usage(arg0);
+        print_usage(stdout, arg0);
+        return EXIT_SUCCESS;
```

Comment:
If we're going to return `EXIT_SUCCESS`, let's add a `help` command which prints to stdout and returns EXIT_SUCCESS. An invalid command should print to stderr and return `EXIT_FAILURE`.

Reply:
While the case is for `CmdInvalid`, this branch only executes when `zig` is called without arguments. I considered renaming it to `CmdNone` for that reason. Invalid commands are handled here: https://github.com/ziglang/zig/blob/master/src/main.cpp#L705-L706

I will add a `help` sub-command since I think it's good practice for any CLI tool. Perhaps, bare `zig` or any other argument error should print an abbreviated usage message pointing the user toward `zig help`?